src/fibonacci.js

```diff
@@ -0,0 +1,14 @@
+export default function fibonacci(n) {
+  const fibStart = [0,1]
+  if (n <= 1){
+    return n
+  } else {
+    function fibBuild(arr, num){
+      if (arr.length === num){
+        return arr
+      } else {
+        return fibBuild(arr.concat([arr[arr.length - 1]+ arr[arr.length-2]]), num)
```

Comment:
This is pretty hard to read as there is a lot going on in this line. I think it would be more readable if you could you capture `arr[arr.length - 1]+ arr[arr.length-2]` in a variable.